src/transformers/generation_utils.py

```diff
@@ -559,6 +562,7 @@ def _get_logits_processor(
         )
         bad_words_ids = bad_words_ids if bad_words_ids is not None else self.config.bad_words_ids
         min_length = min_length if min_length is not None else self.config.min_length
+        max_length = max_length if max_length is not None else self.config.max_length
```

Comment:
he we should also add a 
```
forced_bos_token_id = forced_bos_token_id if forced_bos_token_id is not None else self.config.forced_bos_token_id
```

and add 

`forced_bos_token_id` to the config setting the default to `None`